pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Features
+
+- Operations describe the behaviors and capabilities of the messaging channels described in the AsyncAPI document.
+
+- In a messaging channel, an operation represents a particular action or interaction that can be performed. 
+
+- The purpose of these operations is to provide a standardized means for describing the process of sending, receiving from, requesting, or replying to messages within the messaging system.
+
+## Defining Operations
+
+Operations can be defined as an independent object in the AsyncAPI document. Operations have the following components for it's definition. More information about each field names that are used to define operations can be found [here](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#operationObject). 
+Additionally, an example to show the usage of each field names in defining operations can be found [here](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#operationsObject).
+`operations` are separate objects in the AsyncAPI document on the same level. 
+`channels` can be linked within `operations` by referencing them within the `channels`, just like the following example -
```

Comment:
you have the same in next section

Reply:
sorry, this line was supposed to be removed in the current draft, but for some glitch, it was not removed when I pushed the changes to github this time

Reply:
I have removed the line and pushed the changes again